ScoutSuite/providers/gcp/services/cloudsql.py

```diff
@@ -33,10 +34,12 @@ def parse_instances(self, instance, params):
         instance_dict['name'] = instance['name']
         instance_dict['project_id'] = instance['project']
         instance_dict['automatic_backup_enabled'] = instance['settings']['backupConfiguration']['enabled']
+        instance_dict['database_version'] = instance['databaseVersion']
         instance_dict['log_enabled'] = self._is_log_enabled(instance)
         instance_dict['ssl_required'] = self._is_ssl_required(instance)
-
         instance_dict['backups'] = self._get_instance_backups(instance, params)
+        instance_dict['users'] = self._get_users(instance, params)
+        instance_dict['authorized_networks'] = instance['settings']['ipConfiguration']['authorizedNetworks']
```

Comment:
Is this setting always present in the dictionary or could it throw an error when trying to get it?

Reply:
Would have to test different configurations but if there are no authorized networks I guess it could be missing?

Reply:
I believe I tested this with different configurations and this key is always present in the dictionary.

Reply:
![image](https://user-images.githubusercontent.com/6698529/52576569-500ce700-2dee-11e9-9792-9152768d1de2.png)
Can confirm, it is there even if it's empty.

Reply:
Great! Thanks for checking! :)